src/pages/web-link/web-link.jsx

```diff
@@ -0,0 +1,101 @@
+var React = require('react');
+var render = require('../../lib/render.jsx');
+var TextInput = require('../../components/text-input/text-input.jsx');
+var types = require('../../components/basic-element/basic-element.jsx').types;
+var Platform = require('../../lib/platform.js');
+var LinkedStateMixin = require('react-addons-linked-state-mixin');
+var Alert = require('../../components/alert/alert.jsx');
+
+var java = Platform.getAPI();
+
+var WebLink = React.createClass({
+  mixins: [
+    require('react-intl').IntlMixin,
+    LinkedStateMixin
+  ],
+
+  getInitialState: function () {
+    return {
+      webURL: ''
+    };
+  },
+
+  componentWillMount: function () {
+    this.linkElement = types.link.spec.expand(
+      (JSON.parse(java.getPayloads(`link-element`)))[0].data
```

Comment:
It's generally used for queuing requests to the bulk route of the service. It can also be used to pass data between views.